### PR TITLE
Added test for vlan_range

### LINKED ISSUE #267: Add tests for `vlan_range` epic implementation
`vlan_range` epic implementation is in progress, to ensure it works in a controlled environment, some e2e tests need to be added.

### PR DESCRIPTION
**Pull request by Alopalao**
Closes #267 

### Summary

Added test for `vlan_range` epic:
- Create a circuit with vlan as range
- Patch partial update to circuit.
- Patch fail for different vlan
- Check multiple flows for created circuit
- Check pings with random values within the masks
- Check traces with random values within the masks

### End-To-End tests

```
+ python3 -m pytest tests/ -k 'test_005_evc_vlan_range or test_100_trace_circuit_vlan_range' --reruns 0 -r fEr
============================= test session starts ==============================
platform linux -- Python 3.9.2, pytest-7.2.0, pluggy-1.3.0
rootdir: /tests
plugins: timeout-2.1.0, rerunfailures-10.2, anyio-3.6.2
collected 254 items / 252 deselected / 2 selected

tests/test_e2e_15_mef_eline.py .                                         [ 50%]
tests/test_e2e_40_sdntrace.py .                                          [100%]

```

### CODE REVIEW

**Review by viniarck — COMMENTED** [review]
Great e2e test to cover vlan range use case. 

1) Did you run this locally? 
2) Let's also cover that sdntrace_cp will trace correctly in a new test case in one of the sdntrace_cp test suites.

File: tests/test_e2e_15_mef_eline.py [hunk 1]
```diff
@@ -424,3 +424,86 @@ def test_004_tag_restriction(self):
...
+        flows_s1 = s1.dpctl('dump-flows')
+        flows_s2 = s2.dpctl('dump-flows')
+        assert len(flows_s1.split('\r\n ')) == 8, flows_s1
+        assert 'in_port="s1-eth1",vlan_tci=0x100c/0x1ffc' in flows_s1
+        assert 'in_port="s1-eth1",vlan_tci=0x1010/0x1ffc' in flows_s1
+        assert 'in_port="s1-eth1",vlan_tci=0x1014/0x1ffe' in flows_s1
+        assert len(flows_s2.split('\r\n ')) == 8, flows_s2
+        assert 'in_port="s2-eth1",vlan_tci=0x100c/0x1ffc' in flows_s2
+        assert 'in_port="s2-eth1",vlan_tci=0x1010/0x1ffc' in flows_s2
+        assert 'in_port="s2-eth1",vlan_tci=0x1014/0x1ffe' in flows_s2
```
Comment (viniarck) [inline tests/test_e2e_15_mef_eline.py:509]: Great flows assertion, for completeness, let's also ensure that ping is also working, ideally exercising these three flow entries.

**Comment by viniarck** [comment]
@italovalcy let Aldo know if you'd like to see anything else covered on e2e for vlan range.

File: tests/test_e2e_15_mef_eline.py (hunk 1 above)
Comment (Alopalao) [reply tests/test_e2e_15_mef_eline.py:511]: Added ping

**Review by viniarck — APPROVED** [review]
Nicely done @Alopalao. I'll leave this PR pre-approved.

**Comment by viniarck** [comment]
I'll go ahead and merge this, any extra test case will be covered in a new issue/PR.

**Pull request merged**